opencti-platform/opencti-graphql/tests/02-integration/01-database/middleware-test.js

```diff
@@ -1423,3 +1423,15 @@ describe('Elements deduplication behaviors', () => {
     await deleteElementById(testContext, ADMIN_USER, group2.id, ENTITY_TYPE_THREAT_ACTOR_GROUP);
   });
 });
+
+describe('Delete functional errors behaviors', () => {
+  it('should not be able to delete organization that has members', async () => {
+    await expect(() => deleteElementById(testContext, ADMIN_USER, TEST_ORGANIZATION.id, ENTITY_TYPE_IDENTITY_ORGANIZATION))
+      .rejects.toThrowError('Cannot delete an organization that has members.');
+  });
+  it.skip('should not be able to delete individual associated to user', async () => {
```

Comment:
This test is skipped for now because it looks like admin individual is not yet created. individual-test already tests this, but I wanted to have an extra test here in middleware.